test/autoyast_converter_test.rb

```diff
@@ -93,7 +93,7 @@
     end
   end
 
-  describe ".export" do
+  xdescribe ".export" do
```

Comment:
and I often need various set of configuration and having this stuff in each it, does not look good.
I will try to enable tests where it is possible.